quickblue/src/Explorer/Service.hs

```diff
@@ -11,17 +12,29 @@ import Explorer.Models
 import Explorer.Config
 
 data Explorer f = Explorer
-  { getUspentOutputs :: Gix -> Limit -> f (Items FullTxOut)
+  { getUnspentOutputs        :: Gix -> Limit -> f (Items FullTxOut)
+  , getUnspentOutputsByPCred :: PaymentCred -> Paging -> f (Items FullTxOut)
   }
 
 mkExplorer :: MonadIO f => ExplorerConfig -> Explorer f
-mkExplorer conf = Explorer $ getUnspentOutputs' conf
+mkExplorer conf = Explorer
+  { getUnspentOutputs        = getUnspentOutputs' conf
+  , getUnspentOutputsByPCred = getUnspentOutputsByPCred' conf
+  }
 
 getUnspentOutputs' :: MonadIO f => ExplorerConfig -> Gix -> Limit -> f (Items FullTxOut)
-getUnspentOutputs' ExplorerConfig{..} minIndex limit = do
+getUnspentOutputs' conf minIndex limit =
+  mkGetRequest conf $ "/outputs/unspent/indexed?minIndex=" ++ show minIndex ++ "&limit=" ++ show limit
+
+getUnspentOutputsByPCred' :: MonadIO f => ExplorerConfig -> PaymentCred -> Paging -> f (Items FullTxOut)
+getUnspentOutputsByPCred' conf pcred Paging{..} =
+  mkGetRequest conf $ "/outputs/unspent/byPaymentCred/" ++ show pcred ++  "/?offset=" ++ show offset ++ "&limit=" ++ show limit
```

Comment:
`show pcred`- does it work as you expect? I think it will map param into **"PaymentCred{value}"** but not into **"value"**. It should be `deriving newtype Show` on  PaymentCred

Reply:
same to `offset`

Reply:
Fixed. offset limit are just `Int`s